chart/values.yaml.tmpl

```
@@ -45,6 +45,8 @@ service:
   type: ClusterIP
   port: 3000
 
+privateCAsConfigmap: ~
```

Can we can call it this since the KOTS flag is singular `--private-ca-configmap`?
```suggestion
privateCAConfigmap: ~
```

---

Updated, but it does look weird without hyphens. Which is why it's called `privateCAsConfigmap` everywhere but the command line.